File: CatalogoCervejas.Dominio/Entidades/Receita.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CatalogoCervejas.Dominio.Entidades
{
    public class Receita : EntidadeBase
    {
        public string Garrafa { get; set; }
        public int CervejaId { get; set; }
        public Cerveja Cerveja { get; set; }
        public List<IngredientesReceitas> Ingredientes { get; set; }
    }
}

File: ClientApp/catalogo-cervejas/src/app/models/receita.ts
import { Cerveja } from './cerveja';
import { Ingrediente } from './ingrediente';
import { IngredienteReceita } from './ingredienteReceita';

export class Receita{
    id: number;
    garrafa: string;
    cerveja: Cerveja;
    ingredientes: IngredienteReceita[];

    constructor(){
        this.cerveja = new Cerveja();
    }
}